src/main/starlark/builtins_bzl/common/python/providers.bzl

```diff
@@ -14,6 +14,7 @@
 """Providers for Python rules."""
 
 DEFAULT_STUB_SHEBANG = "#!/usr/bin/env python3"
+DEFAULT_BOOTSTRAP_TEMPLATE = "@bazel_tools//tools/python:python_bootstrap_template.txt"
```

Comment:
I've landed a change in the last day that probably means you'll need to merge and resolve. When merging, please load semantics.bzl here and use the TOOLS_REPO constant to construct the "@baze_tools" portion of these location strings.